### ISSUE TITLE
Fatal Error: Uncaught exception "PharExcepetion"

### ISSUE THREAD

**Issue by richcaputo** [BrunoDeBarros/git-deploy-php#70]
When I run php git-deploy I keep getting this error:

`Fatal error: Uncaught exception 'PharException' with message 'phar "/Users/RichardCaputo/GitHub/sst-dashboard/git-deploy" SHA1 signature could not be verified: broken signature' in /Users/RichardCaputo/GitHub/sst-dashboard/git-deploy:5
Stack trace:
#0 /Users/RichardCaputo/GitHub/sst-dashboard/git-deploy(5): Phar::mapPhar('git-deploy')
#1 {main}
  thrown in /Users/RichardCaputo/GitHub/sst-dashboard/git-deploy on line 5`

**Comment by BrunoDeBarros** [comment]
Ooh, that's really odd. I'll have a look at it. Which PHP version are you using? Not that I think it makes a difference, but maybe it does.

**Comment by richcaputo** [comment]
It is running on AWS EC2 - PHP Version 5.6.13

If I understand your readme correctly: I only need the git-deploy (on root) and then setup my ftp credentials in the initial.ini (on root) and then run php git-deploy from command line

**Comment by BrunoDeBarros** [comment]
Yes, everything should work fine. Sounds like the git-deploy file is corrupted, from your error message. I'm going to try and clone the repo and try and see if there's anything wrong with it.

**Comment by richcaputo** [comment]
If I look at the file in Sublime, I see sections like this: 

![screen shot 2016-06-24 at 2 51 38 pm](https://cloud.githubusercontent.com/assets/14776792/16347531/2e90e15e-3a1b-11e6-87c4-e7e49b4f6f29.png)

**Comment by BrunoDeBarros** [comment]
That's expected; it's a zipped Phar file so it makes sense that it's unreadable.

**Comment by richcaputo** [comment]
Ok. Let me know if you find anything out, I'm anxious to get this working. Thanks for you help.

**Comment by BrunoDeBarros** [comment]
I've just tried it and had no problems; is it possible that the file's been corrupted or maybe changed by Git, or something (on Windows, you have that whole LF to CRLF thing, and FTP/SFTP clients do that too sometimes, so if you uploaded with FTP, that might be the issue)?

If it is indeed something like that, you might be able to resolve it by adding this to your php.ini:

```
phar.require_hash=0
```

The error you got indicates that the file you have has been modified. It seems to be fine when I make a clean clone, and it seems to be fine for the hundreds of people who are using it and have starred it, so I'm hoping you'll be able to fix it with the above php.ini change.
